frontends/web/src/api/accountsync.ts

```diff
@@ -30,6 +30,18 @@ export const syncAccountsList = (
   return subscribeEndpoint('accounts', cb);
 };
 
+/**
+ * Returns a function that subscribes a callback on an "account/<CODE>/synced-addresses-count"
+ * event to receive the progress of the address sync.
+ * Meant to be used with `useSubscribe` or `useSubscribeMap`.
+ */
+export const subscribeAddressesCount = (code: AccountCode) => (
+  (cb: TSubscriptionCallback<number>) => {
+		 return subscribeEndpoint(`account/${code}/synced-addresses-count`, (
+      count: number,
+    ) => cb(count));
+  });
```

Comment:
I strongly thin we shouldn't duplicate endpoints and have them return different function styles. Maybe the way to go forward is to rewrite `syncAddressCount` calls to `subscribeAddressesCount` everywhere, but see my other comment first.

Reply:
I took your version, as it is most flexible and rewrote the other palce where this function is used. in https://github.com/digitalbitbox/bitbox-wallet-app/pull/2211

Reply:
I'll close this